### ISSUE TITLE
Geeqie flickers during of loading big gif image

### ISSUE THREAD

**Issue by jekhor** [BestImageViewer/geeqie#967]
### ISSUE TYPE
* Bug Report

### GEEQIE VERSION
```
Geeqie 1.7.2 GTK3
```
### OS / DISTRIBUTION
Debian sid (unstable), geeqie 1:1.7.2-1+b1 amd64

### SUMMARY
When trying to open big gif file (attached), geeqie starts to flicker showing only fragment of image. After few minutes image is loaded and then can be viewed as expected (with zoom/panning working).  I recorded a screencast demonstrated this.

https://user-images.githubusercontent.com/117467/153765366-281e7e8b-7557-490c-9d65-5d50f4bb778b.mp4

![c00002-A-014-035-058](https://user-images.githubusercontent.com/117467/153765383-d839c9db-8741-47aa-adae-8a372be77f3a.gif)

### STEPS TO REPRODUCE
Try to open attached gif.

**Comment by caclark** [comment]
I do not think that the size is a problem, but that it is malformed in some way.
There is text at the end of the file that should not be there. 
Other viewers can display the image, but I do not think it worth  pursuing this unless the problem occurs elsewhere.

**Comment by caclark** [comment]
> Moreover, the same file was displayed properly by geeqie in the past.

Which version of Geeqie was working? For me, the image fails on all versions from v1.3 to v1.7

**Comment by jekhor** [comment]
I cannot remember, it was 5-10 years ago maybe :)
